src/entities/Proposal/templates/addPoi.ts

```diff
@@ -1,7 +1,10 @@
+import useFormatMessage from "decentraland-gatsby/dist/hooks/useFormatMessage";
 import Land from "decentraland-gatsby/dist/utils/api/Land";
 import { NewProposalPOI } from "../types";
 import { formatMarkdown, template } from "./utils";
 
+// const l = useFormatMessage();
+
 export const title = (proposal: NewProposalPOI) =>
```

Comment:
This is where a the `poiType` field in the `NewProposalPOI` would also be useful. the `addPoi.ts` file could contain all the logic needed for generating either template

Reply:
The comment here is because I tried to replace the hardcoded strings in the template (to be consistent with i18n), but when I added that line I got a css error that didn't make any kind of sense and I didn't know how to fix it. I left the comment to ask you later.